src/test/java/gov/usgs/owi/nldi/controllers/CharacteristicsControllerTest.java

```diff
@@ -0,0 +1,80 @@
+package gov.usgs.owi.nldi.controllers;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+import gov.usgs.owi.nldi.dao.LookupDao;
+import gov.usgs.owi.nldi.dao.StreamingDao;
+import gov.usgs.owi.nldi.services.LogService;
+import gov.usgs.owi.nldi.services.Navigation;
+import gov.usgs.owi.nldi.services.Parameters;
+import gov.usgs.owi.nldi.springinit.TestSpringConfig;
+
+public class CharacteristicsControllerTest {
+
+	private StreamingDao streamingDao;
+	@Mock
+	private LookupDao lookupDao;
+	@Mock
+	private Navigation navigation;
+	@Mock
+	private Parameters parameters;
+	@Mock
```

Comment:
I like this @Mock annotation! It's much cleaner than what I had been doing: `Mockito.mock(MyClass.class)`. Thanks for the hot tip!